### PR TITLE
fix: logging exception for ill-formed NC messages

### PR DESCRIPTION
**Pull request by wburkhardt**
change: Logging exceptions raised by the pretty formatter are caught,
and an error message and the plain NC message are logged instead.
speed: A little bit more speed improvement (long messages).

### CODE REVIEW

File: netconf_client/ncclient.py [hunk 1]
```diff
@@ -32,9 +32,17 @@
...
 
-    parser = etree.XMLParser(remove_blank_text=True)
-    tree = etree.fromstring(xml, parser)
-    return etree.tostring(tree, pretty_print=True).decode()
+    pretty = ""
+    try:
+        parser = etree.XMLParser(remove_blank_text=True)
+        tree = etree.fromstring(xml, parser)
+        pretty = etree.tostring(tree, pretty_print=True).decode()
+    except Exception as e:
```
Comment (Nitori-) [inline netconf_client/ncclient.py:40]: This needs to catch the specific type of exception

Comment (wburkhardt) [reply netconf_client/ncclient.py:40]: Yes, good point. Would it be sufficient to catch the lxml.etree **base class** [Error](https://lxml.de/api/lxml.etree.Error-class.html)? And any other exception types (not derived directly or indirectly from Error) remain unhandled?

Comment (Nitori-) [reply netconf_client/ncclient.py:40]: Yes I think that would work

Comment (wburkhardt) [reply netconf_client/ncclient.py:40]: Yes, it works fine. The unit-tests run successfully with this change. I re-submit...

**Pull request merged**